src/App.less

```diff
@@ -11,3 +11,11 @@ body {
 .seksjon {
   padding-bottom: 3rem;
 }
+
```

Comment:
La til disse generelle css-klassene i App.less fordi jeg hadde behov for å bare legge på litt margin på bunnen noen steder, men kanskje det heller kan gjøres inline? Hva tenker dere?

Reply:
Hadde vår styled component` <SeksjonGruppe/>` oppfylt dette behovet? Hvis du bruker denne til å wrappe der du trenger litt padding på bunnen?

Reply:
Denne var her allerede (mulig denne også kan byttes ut, men den brukes andre steder), men jeg tenkte på de under, som legger til margin på 0.5rem og 1rem.


margin-bottom: 0.5rem brukes for å legge til mellomrom mellom spørsmålet og tekstfeltet her: ![Screen Shot 2020-05-15 at 3 17 19 PM](https://user-images.githubusercontent.com/1413265/82054465-48a64e80-96bf-11ea-8e65-bcc2059654d8.png)

Reply:
ahh, hmm. Er det bare her? Eller brukes marginen flere steder? 
Funker `<FeltGruppe/>`?  Den har `25px` padding-bottom, med mindre det måå være margin.

Reply:
Fikk fjernet klassen her: 9673cb7